modules/core/file_fetcher.js

```diff
@@ -29,24 +34,23 @@ export function coreFileFetcher() {
     'qa_data': 'data/qa_data.min.json',
     'shortcuts': 'data/shortcuts.min.json',
     'territory_languages': 'data/territory_languages.min.json',
-    'oci_defaults': ociCdnUrl.replace('{version}', ociVersionMinor) + 'dist/defaults.min.json',
-    'oci_features': ociCdnUrl.replace('{version}', ociVersionMinor) + 'dist/featureCollection.min.json',
-    'oci_resources': ociCdnUrl.replace('{version}', ociVersionMinor) + 'dist/resources.min.json',
-    'presets_package': presetsCdnUrl.replace('{presets_version}', presetsVersion) + 'package.json',
+    'oci_defaults': ociCdnUrl + 'dist/defaults.min.json',
+    'oci_features': ociCdnUrl + 'dist/featureCollection.min.json',
+    'oci_resources': ociCdnUrl + 'dist/resources.min.json',
+    'presets_package': presetsCdnUrl + 'package.json',
     'deprecated': presetsCdnUrl + 'dist/deprecated.min.json',
     'discarded': presetsCdnUrl + 'dist/discarded.min.json',
     'preset_categories': presetsCdnUrl + 'dist/preset_categories.min.json',
     'preset_defaults': presetsCdnUrl + 'dist/preset_defaults.min.json',
     'preset_fields': presetsCdnUrl + 'dist/fields.min.json',
     'preset_presets': presetsCdnUrl + 'dist/presets.min.json',
```

Comment:
These files (except `presets_package`) should use the `presetsCdnUrlTemplate` instead of ~~`presetsCdnUrlTemplate`~~. See my comment [below](https://github.com/openstreetmap/iD/pull/9475/commits/1d64839b24d9c7f6d79b3810961d1f0a27c59d27#diff-849c38d4d2f9a07e9301d615da477fc56d407fca11d991fcccd50ba96a620fd5R93).

Reply:
For example:

```suggestion
    'deprecated': (presetsUrlOverwrite || presetsCdnUrlTemplate) + 'dist/deprecated.min.json',
    'discarded': (presetsUrlOverwrite || presetsCdnUrlTemplate) + 'dist/discarded.min.json',
    'preset_categories': (presetsUrlOverwrite || presetsCdnUrlTemplate) + 'dist/preset_categories.min.json',
    'preset_defaults': (presetsUrlOverwrite || presetsCdnUrlTemplate) + 'dist/preset_defaults.min.json',
    'preset_fields': (presetsUrlOverwrite || presetsCdnUrlTemplate) + 'dist/fields.min.json',
    'preset_presets': (presetsUrlOverwrite || presetsCdnUrlTemplate) + 'dist/presets.min.json',
```